oereb_belastete_standorte/build.gradle

```diff
@@ -42,7 +36,7 @@ task downloadCantonalLegalBasis(type: Download, dependsOn: "downloadFederalLegal
 
 task downloadKbs(type: Download, dependsOn: "downloadCantonalLegalBasis"){
     description = "Download Belastete Standorte"
-    src "https://data.geo.admin.ch/" + kbsDataSet + "/data.zip"
+    src "http://altlast4web-t.verw.rootso.org/interlis_exports/oereb/data.zip"
```

Comment:
Das müssen wir mit einer Variablen lösen, damit die Prod.-Umgebung dann auf den produktiven Altlast4web-Server geht.

Ich werde in GRETL eine Variable `ORG_GRADLE_PROJECT_altlast4webHostName` setzen; auf diese kannst du im build.gradle mit `altlast4webHostName` zugreifen. Der Wert der Variablen wird sein:

* `altlast4web-t.verw.rootso.org` in GRETL Test und GRETL Integration
* `altlast4web.verw.rootso.org` in GRETL Produktion

Reply:
Wir machen dies nach Besprechung doch anders: GRETL soll aus allen drei Umgebungen immer auf den produktiven Altlast4Web-Server zugreifen. Also kann die URL einfach fix ins build.gradle geschrieben werden. Und es braucht die Variablen nicht.